<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="10" guice-stage="DEVELOPMENT" name="Failed suite [Suite]" parallel="methods">
  <test name="Test2(failed)" parallel="methods">
    <classes>
      <class name="com.w2a.archBatch2.TestCases.ValidateFacebookLogin">
        <methods>
          <include name="beforeClass"/>
          <include name="validateFBLoginWIthValidCredentials"/>
          <include name="beforeMethod"/>
          <include name="afterSuite"/>
          <include name="afterTest"/>
          <include name="beforeSuite"/>
          <include name="afterClass"/>
          <include name="tc_02"/>
          <include name="beforeTest"/>
          <include name="afterMethod"/>
        </methods>
      </class> <!-- com.w2a.archBatch2.TestCases.ValidateFacebookLogin -->
      <class name="com.w2a.archBatch2.TestCases.TestCase1">
        <methods>
          <include name="beforeClass"/>
          <include name="beforeMethod"/>
          <include name="afterSuite"/>
          <include name="afterTest"/>
          <include name="beforeSuite"/>
          <include name="afterClass"/>
          <include name="tc_03"/>
          <include name="beforeTest"/>
          <include name="afterMethod"/>
        </methods>
      </class> <!-- com.w2a.archBatch2.TestCases.TestCase1 -->
    </classes>
  </test> <!-- Test2(failed) -->
</suite> <!-- Failed suite [Suite] -->
